embulk-core/src/main/java/org/embulk/exec/StdoutReporterPlugin.java

```diff
@@ -0,0 +1,49 @@
+package org.embulk.exec;
+
+import org.embulk.config.ConfigSource;
+import org.embulk.config.Task;
+import org.embulk.config.TaskSource;
+import org.embulk.spi.AbstractReporterImpl;
+import org.embulk.spi.ReporterPlugin;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.util.Map;
+
+// TODO should describe why we need to implement this plugin here.
+public class StdoutReporterPlugin
+        implements ReporterPlugin
+{
+    @Override
+    public TaskSource configureTaskSource(final ConfigSource config)
+    {
+        return config.loadConfig(Task.class).dump();
+    }
+
+    @Override
+    public AbstractReporterImpl open(final TaskSource task)
+    {
+        return new StdoutReporterImpl();
+    }
+
+    @ThreadSafe
+    private static class StdoutReporterImpl
+            extends AbstractReporterImpl
+    {
+        @Override
+        public void report(Level level, Map<String, Object> event)
+        {
+            System.out.println(event);
```

Comment:
While it can be later, we may want a kind of pretty formatting, sorting by key names, or else. Can you just add a TODO comment?